core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -596,11 +597,12 @@ oppia.directive('conversationSkin', [
                       focusService.setFocusIfOnDesktop(_nextFocusLabel);
                       scrollToBottom();
                     } else {
-                      playerTranscriptService.addNewResponse(feedbackHtml);
+                      PlayerTranscriptService.addNewResponse(feedbackHtml);
                       $scope.showPendingCard(
                         newStateName,
                         newParams,
-                        contentHtml + oppiaPlayerService.getRandomSuffix());
+                        contentHtml +
+                        ExplorationPlayerService.getRandomSuffix());
```

Comment:
Sorry I didn't made it very clear last time, could you format it like this:
 
![indent](https://user-images.githubusercontent.com/7807557/30772872-ad8f9f76-a082-11e7-9d76-918cb1bad283.jpeg)